pkg/secrets/aws_test.go

```diff
@@ -0,0 +1,87 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	. "gopkg.in/check.v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { TestingT(t) }
```

Comment:
[minor] usually we register the tests in a file with the same name as the package.